### ISSUE TITLE
Contributor Style Guide

### ISSUE THREAD

**Issue by wonderchook** [hotosm/learnosm#82]
With the desire from so many people to add content we really need a style guide. This will ensure consistent formatting and voice in LearnOSM.

@MappingKat and @jeffhaack do one of you have time to lead this? Let's put ideas of things it needs to contain in this ticket.

**Comment by Nick-Tallguy** [comment]
I'm aware that hackpad is disappearing at some point. I've copied all of the text from https://hackpad.com/Proposed-Style-Guide-for-LearnOSM-1JJxS8NIQSX with a view to using this in a style guide

**Comment by harry-wood** [comment]
So to be clear, the style guide is now part of the learnosm website itself at:

https://learnosm.org/en/contribute/style/   ([in github here](https://github.com/hotosm/learnosm/blob/gh-pages/_posts/en/0150-10-20-style_en.md)) 

Although it still feels a bit casually pasted in. It fact it says _"Paste of data from hackpad"_ at the top (The hackpad no longer exists). And it's not added to the side navigation. I found it by googling.

The above discussion suggests it was never exactly declared "final and ready to publish", but then I would suggest that's not an appropriate hoop to make ourselves jump through at this point. It was linked to from CONTRIBUTING.md for a few years (then discussion because it disappeared from hackpad). I'd suggest we regard it as "published but obviously never finished", meaning we can close this issue and make new issues for the tidying that needs to happen.

**Comment by michael63-osm** [comment]
I agree that this needs to be added to the information available from https://learnosm.org/en/contribute/ as mentioned in #651.
